tests/FakeItEasy.Specs/IllegalFluentApiSpecs.cs

```diff
@@ -147,15 +147,8 @@ public static void IllegalSyntax(string illegalOperation, string expectedError,
                 .x(() => diagnostics.Where(d => d.Severity == DiagnosticSeverity.Error).Should().ContainSingle()
                     .Which.GetMessage().Should().Contain(expectedError));
 
-#if MISMATCH_COMPILE_AND_RUNTIME_FRAMEWORKS
-            "And there are warnings due to a mistmach between compile-time and runtime frameworks"
-                .x(() => diagnostics.Where(d => d.Severity != DiagnosticSeverity.Error).Should().OnlyContain(d =>
-                    d.Severity == DiagnosticSeverity.Warning &&
-                    d.Id == "CS1701"));
-#else

```

Comment:
I don't remember this. Was this an error that only occurred in .NET 6.0?

Reply:
Yeah. Because we built with 5.0 and tested with 6.0.